install/kubernetes/cilium/charts/hubble-tls/templates/secret.yaml

```diff
@@ -45,15 +45,31 @@ data:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: hubble-relay-certs
+  name: hubble-relay-client-certs
```

Comment:
One question for my understanding, seems like any time helm (e.g. upgarde with re-use values) will generate the new values for certs, will hubble-relay get notified and restart ?

Reply:
This is a good point. I don't think Hubble Relay will get notified. The best approach to address this is probably to watch for cert files changes (typically via fsnotify) and reload the certs when this happens.
There shouldn't be any reason to restart pods when new certs are generated as there is no point breaking existing connections (it's not a problem if certs are re-generated while connections are active because the TLS certs are only useful for the TLS handshake during which 4 session keys are exchanged to encrypt the communication).
I plan to work on this in a follow-up PR. This one is already pretty big.